### PR TITLE
Fix empty telegram handle after @

### LINKED ISSUE #176: [PE-D][Tester F] Able to create a student with an empty telegram handle
- My input for the `h/TELEGRAM_HANDLE` input was `h/@`

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: yongkotaro/ped#1

### PR DESCRIPTION
**Pull request by matthewken19**
Fix #176

### CODE REVIEW

**Comment by tanjiajiajun** [comment]
Should be fixed in PR #254 already

File: src/main/java/educonnect/model/student/TelegramHandle.java [hunk 1]
```diff
@@ -10,8 +10,9 @@
 public class TelegramHandle {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Telegram handles should start with @";
-    public static final String VALIDATION_REGEX = "^@.*";
+            "Telegram handles should start with '@' and "
+                    + "contains at least 1 alphanumeric characters and/or underscores after it.";
+    public static final String VALIDATION_REGEX = "^@[a-zA-Z0-9_]+$";
```
Comment (jeremiahlzz) [inline src/main/java/educonnect/model/student/TelegramHandle.java:15]:
```suggestion
    public static final String VALIDATION_REGEX = "^@\\w+$";
```
A more concise way of achieving the same thing, `\w` regex is short form for `[a-zA-Z0-9_]`.
Explicitly showing a-zA-Z may be better for people less familiar with regex.

**Review by jeremiahlzz — APPROVED** [review]
LGTM

**Comment by matthewken19** [comment]
> Should be fixed in PR #254 already

Oh my bad, I didn't check your PR properly. I'll leave it here in case there is anything you want to check. Can just close it once you don't need it.

File: src/main/java/educonnect/model/student/TelegramHandle.java (hunk 1 above)
Comment (marclamp) [reply src/main/java/educonnect/model/student/TelegramHandle.java:15]: agreed, I had to google \w hahaha

**Pull request closed without merging**